test/integration/resources/init.c

```diff
@@ -0,0 +1,148 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+/*
+ * Copyright (C) 2020 embedded brains GmbH (http://www.embedded-brains.de)
```

Comment:
How much of their code do we have? I see mostly the generic RTEMS bootstrap, which I can't imagine to be copyrightable...